client/src/components/ToDoList/useTodosState.js

```diff
@@ -6,20 +6,28 @@ const saveListToStorage = (list) => {
     localStorage.setItem("todoList", listJson);
 };
 
+const saveTaskPointsToStorage = (taskPoints) => {
+    localStorage.setItem("taskPoints", taskPoints);
+};
+
 const sortOverallTodosByDate = (overallTodos) => {
     let sorted = {};
-    Object.keys(overallTodos).sort().forEach(date => {
-        sorted[date] = overallTodos[date];
-    });
+    Object.keys(overallTodos)
+        .sort()
+        .forEach((date) => {
+            sorted[date] = overallTodos[date];
+        });
     return sorted;
 };
 
-const sortSpecificTodoListByTime = (todos) => todos.sort((thisTodo, otherTodo) => thisTodo.time.localeCompare(otherTodo.time));
+const sortSpecificTodoListByTime = (todos) =>
+    todos.sort((thisTodo, otherTodo) => thisTodo.time.localeCompare(otherTodo.time));
 
 const newTodaysDate = moment().format("YYYY-MM-D");
 
 function useTodosState(initialTodos, setIsAddingItem) {
     const [todoList, setTodoList] = useState(initialTodos);
+    const [taskPoints, setTaskPoints] = useState(0);
```

Comment:
I was aware of this issue but at 2am last night I said "good enough" HAHA. I will try do this now.

Reply:
Do you think that I should, as I have done with task points, weave the saving of `progress` into your useTodosState? Or do you think it would be fine having it inside the Plant game? I think the plant progress is unrelated to the Todo list, so maybe inside the plant game is fine.

Reply:
Yeah go ahead with saving the `progress` inside the plant game and see if it works